### ISSUE TITLE
AJAX calls are intercepted.

### ISSUE THREAD

**Issue by neon5** [panique/php-login#554]
Using the Advanced version I need to do an ajax get when a select control is changed so that I can populate its related select control.

However, I can't seem to just get back only the data.  The framework is returning an entire HTML page with header, body tags and all. If I put a breakpoint I can see that the framework is running the application load instead of just hitting the page I am pointing the ajax url to.  Maybe it is due to the .htaccess file -- I don't know.

So can someone tell me how I can do an ajax call with this framework doing something simple like this?
```
$("#User").change(function() {
    $.ajax({
        url: "getcurrentregions.php",
        data: { user_id: $(this).val() },
        type: "GET"
    }).done(function( results ) {
        $("#Messages").html(results);
    });
});
```
Thank you.

**Comment by panique** [comment]
This is php-login, not php-login-advanced.
If you want to do ajax calls, then build this by yourself, and simply do not deliver header, footer and page content. It's totally obvious.

**Comment by neon5** [comment]
Hi panique,

This framework is very advanced to me and I've learned a lot from it.  It has help me tremendously with a site I am working on and I tell everyone about php-login.

I did create the ajax call and the listening page myself.  Which I've done before.  I created the page to return nothing but the data.  But my page is getting intercepted, possibly due to the .htaccess file, and application.php is being called.

I was just hopping someone might know why this is happening but I think it's too advance of a question.

Thanks though.
